### ISSUE TITLE
Review list of topics and provide feedback

### ISSUE THREAD

**Issue by stephwright** [mozillascience/open-data-training#22]
The current list of topics and delivery methods is here: https://github.com/mozillascience/open-data-training/blob/master/planning/topics.md

Would love thoughts regarding topics we haven't considered, particularly level 3 and thoughts for what should be included in disciplinary items in Level 2.

Also interested in ideas for other delivery methods separate from modules & primers.  Ideas for videos perhaps?  online courses?

**Comment by amelgh** [comment]
Hi Steph,

Great content Steph!
- I thought, it would be great to include some real examples in level 1 to highlight the advantages of open data versus non open data 
- It would be great to focus more on Ethics aspects in Level 3. 
- What about adding different possible/classical data analysis pipelines to stress on the fact that data could be analyzed in different ways?

A great paper that might be useful and added for further readings: http://www.nature.com/ngeo/journal/v7/n11/full/ngeo2294.html

Amel

**Comment by Dhaferl** [comment]
Hi,
I guess IRB stands for Institutional Review Board. As far as the Life Sciences and even Social sciences might include data on animals, environment, etc…, it would perhaps be better to add IRB issue also to these topics
Dhafer

**Comment by amelgh** [comment]
A great paper that might be useful : http://www.nature.com/ngeo/journal/v7/n11/full/ngeo2294.html

**Comment by SiVv-B** [comment]
Hi there,
We went through the different primers and many of us think that it would make more sens to have primer 3 before primer 2

**Comment by zbouslama** [comment]
I suggest that in the begginning of primer 1, you could add a short text about how much open data is involved in our everyday's life and doesn't present a threat to us to share data. In fact, users nowadays are not only "data lovers", but  everyone could be an open data user for a short or a long period.

**Comment by stephwright** [comment]
So grateful to the entire Tunis crew for your feedback on this!!!
@amelgh I've added your feedback into the topics list
@Dhaferl Apologies, yes! IRB = Institutional Review Board and you're right it should be added to Life Sci & Soc Sci.  I've included that in the list
@Siwar-BLK I'm going to sit down w/ @zee-moz next week and rethink the order of these.  Thank you!
@zbouslama thank you! We really hadn't addressed as much the "everyone" can do it aspect.  I like the idea of addressing it in a short para at the front

**Comment by JEK-III** [comment]
- Social sciences definitely needs a privacy/confidentiality unit. IRB could be folded in to that. The ICPSR guidelines for writing informed consent forms so that you'll be able to share the data later would be a really good resource for that section. 
- Physical sciences could have a unit on dynamic and/or very large datasets. 
- Maybe this is what you have in mind for 'publishing considerations,' but Humanities could have a unit on datasets composed of copyrighted material. 

FWIW, I like the current order of the primers. If some of the content from primer 3 needs to be moved up, I could also imagine creating a primer 1.5 that would be something like "How to collect shareable/reuseable data" that would have a bit about the metadata people will need and the "Can/should I share?" section from Primer 3 recast in terms of what to do upfront if you want to be able to legally & ethically share your data later. The occasional forward reference to things not covered yet is probably inevitable and really no big deal.

**Comment by stephwright** [comment]
Thanks @JEK-III !!! 
I've added all that to the topics list.
I agree that we'll probably have some forward referencing going on but all the feedback on ordering is super helpful.

**Comment by ajspadial** [comment]
I'm not sure if it's included under "things to know before using other people's data", but I'd add "Quoting open data" in level 1.

**Comment by stephwright** [comment]
Absolutely... thanks @ajspadial. Seems like Primer 5 is a good place to put that.  Thanks for the feedback!
